src/components/NavBar.vue

```diff
@@ -129,11 +129,17 @@ export default {
           cursor: pointer;
         }
 
-        .router-link-exact-active,
-        :hover {
+        .router-link-active {
```

Comment:
Cool ! However you should use mixin in this case to avoid css duplication : 

```
       @mixin activatedLink() {
          margin-bottom: -2px;
          border-bottom: 2px solid $vue-color;
        }

        .router-link-active {
          @include activatedLink;
        }

        @media (hover) {
          :hover {
            @include activatedLink;
          }
        } 
```

Reply:
That's a good idea ! I do it right now !